src/main/java/seedu/address/logic/commands/RemoveTagCommand.java

```diff
@@ -49,19 +60,61 @@ public CommandResult executeUndoableCommand() throws CommandException {
         requireNonNull(model);
         try {
             if (index.orElse(null) == null) {
-                model.removeTag(toRemove);
```

Comment:
I recommend putting a try block into both the `if` block and the `else` block,
so that only the possible exceptions generated from the two uses of `removetag` command gets handled.

It is better to have more precise exception handling, and shows that you have a clear idea what can go wrong for each of the different usage of `removetag` command.